diff --git a/dist/module.d.mts b/dist/module.d.mts
index 5c1248cdb7061f51e7e57026c6742e90bf6e0c1f..d2d211af091230b75be78670725de56971f973e9 100644
--- a/dist/module.d.mts
+++ b/dist/module.d.mts
@@ -34,7 +34,8 @@ interface RouteLocation {
 interface MenuLinkRoute {
   label: string;
   icon?: string;
-  to?: string | RouteLocation;
+  // CHANGED: Otherwise we get TS errors
+  to?: any;
   /** @deprecated since version 1.11.0, instead use `to` */
   path?: string | RouteLocation;
   children?: MenuLinkRoute[];
diff --git a/dist/module.d.ts b/dist/module.d.ts
index 8683c8461a3ebbfec663ad5bfd14740ec6df927a..333363f02c995a448307c53447649daf05d0fe3c 100644
--- a/dist/module.d.ts
+++ b/dist/module.d.ts
@@ -34,7 +34,8 @@ interface RouteLocation {
 interface MenuLinkRoute {
   label: string;
   icon?: string;
-  to?: string | RouteLocation;
+  // CHANGED: Otherwise we get TS errors
+  to?: any;
   /** @deprecated since version 1.11.0, instead use `to` */
   path?: string | RouteLocation;
   children?: MenuLinkRoute[];
diff --git a/dist/runtime/types/index.d.ts b/dist/runtime/types/index.d.ts
index 64a4f8cf37643ee23bfbf151144f88c2924e32ab..286d95d67bcfce7131476f3cee772ce4a7bebda3 100644
--- a/dist/runtime/types/index.d.ts
+++ b/dist/runtime/types/index.d.ts
@@ -33,7 +33,8 @@ interface RouteLocation {
 export interface MenuLinkRoute {
   label: string;
   icon?: string;
-  to?: string | RouteLocation;
+  // CHANGED: Otherwise we get TS errors
+  to?: any;
   /** @deprecated since version 1.11.0, instead use `to` */
   path?: string | RouteLocation;
   children?: MenuLinkRoute[];
@@ -46,7 +47,7 @@ export type ColorModePreference = "light" | "dark" | "system";
 export type ColorModeForce = ColorMode | false;
 
 export interface PublicConfig {
-   /** @deprecated since version 1.12.0, instead use `naiveui.themeConfig` in `app.config` */
+  /** @deprecated since version 1.12.0, instead use `naiveui.themeConfig` in `app.config` */
   themeConfig?: ThemeConfig;
   colorModePreference: ColorModePreference;
   colorModePreferenceCookieName: string;
