diff --git a/dist/runtime/components/NaiveIcon.vue b/dist/runtime/components/NaiveIcon.vue
index 52f2327bdfd2e2cc880fe5cdb03b007579411952..f05a210e249b0ad894e7b191c4dd5d930b8e6e36 100644
--- a/dist/runtime/components/NaiveIcon.vue
+++ b/dist/runtime/components/NaiveIcon.vue
@@ -1,5 +1,5 @@
 <template>
-    <n-icon-wrapper :size="sSize" :border-radius="borderRadius" :color="color" :icon-color="iconColor">
+    <n-icon-wrapper v-if="icon" :size="sSize" :border-radius="borderRadius" :color="color" :icon-color="iconColor">
         <Icon :icon="icon" :width="sSize" :height="sSize" />
     </n-icon-wrapper>
 </template>
diff --git a/dist/runtime/components/NaiveNavbar.vue b/dist/runtime/components/NaiveNavbar.vue
index 56254eccf8b3eed296abb359fe2696727158bc38..e36d5ab5887b1d204c43c46400c92f9ad3d73eca 100644
--- a/dist/runtime/components/NaiveNavbar.vue
+++ b/dist/runtime/components/NaiveNavbar.vue
@@ -117,6 +117,7 @@ const menuOptions = computed<MenuOption[]>(() => {
         const menuOption: MenuOption =
         {
             label: route.path ? () => h(NuxtLink, { to: route.path }, { default: () => route.label }) : route.label,
+            // @ts-ignore: https://github.com/becem-gharbi/nuxt-naiveui/issues/18
             icon: route.icon ? () => h(NaiveIcon, { name: route.icon }) : undefined,
             key: route.path || route.label,
         }
