diff --git a/dist/runtime/components/Prose/ProseH1.vue b/dist/runtime/components/Prose/ProseH1.vue
index 6381a794df6642a5deed58d30217aacf98da7759..61125e2fa38e0baff8039307202a022368762e4f 100644
--- a/dist/runtime/components/Prose/ProseH1.vue
+++ b/dist/runtime/components/Prose/ProseH1.vue
@@ -12,5 +12,6 @@ import { useRuntimeConfig } from '#imports'
 defineProps<{ id?: string }>()
 const heading = 1
 const { anchorLinks } = useRuntimeConfig().public.content
+// @ts-expect-error: https://github.com/nuxt/content/issues/1856
 const generate = anchorLinks?.depth >= heading && !anchorLinks?.exclude.includes(heading)
 </script>
diff --git a/dist/runtime/components/Prose/ProseH2.vue b/dist/runtime/components/Prose/ProseH2.vue
index c3d2417f44992c7b53b4cf75a07ee51f59734de1..62427a423ea699fab2eab16ef829a194810f9a18 100644
--- a/dist/runtime/components/Prose/ProseH2.vue
+++ b/dist/runtime/components/Prose/ProseH2.vue
@@ -12,5 +12,6 @@ import { useRuntimeConfig } from '#imports'
 defineProps<{ id?: string }>()
 const heading = 2
 const { anchorLinks } = useRuntimeConfig().public.content
+// @ts-expect-error: https://github.com/nuxt/content/issues/1856
 const generate = anchorLinks?.depth >= heading && !anchorLinks?.exclude.includes(heading)
 </script>
diff --git a/dist/runtime/components/Prose/ProseH3.vue b/dist/runtime/components/Prose/ProseH3.vue
index 9079b849990cda697ab63e21f5d90373e0f4393e..5e928d509f388109b42cadbd2b918a16243df604 100644
--- a/dist/runtime/components/Prose/ProseH3.vue
+++ b/dist/runtime/components/Prose/ProseH3.vue
@@ -12,5 +12,6 @@ import { useRuntimeConfig } from '#imports'
 defineProps<{ id?: string }>()
 const heading = 3
 const { anchorLinks } = useRuntimeConfig().public.content
+// @ts-expect-error: https://github.com/nuxt/content/issues/1856
 const generate = anchorLinks?.depth >= heading && !anchorLinks?.exclude.includes(heading)
 </script>
diff --git a/dist/runtime/components/Prose/ProseH4.vue b/dist/runtime/components/Prose/ProseH4.vue
index be8d1a1da9b692a6531d4111681ace75c7512614..8c5a97cd246fdb3cab56a8008dbb26afbce615b2 100644
--- a/dist/runtime/components/Prose/ProseH4.vue
+++ b/dist/runtime/components/Prose/ProseH4.vue
@@ -12,5 +12,6 @@ import { useRuntimeConfig } from '#imports'
 defineProps<{ id?: string }>()
 const heading = 4
 const { anchorLinks } = useRuntimeConfig().public.content
+// @ts-expect-error: https://github.com/nuxt/content/issues/1856
 const generate = anchorLinks?.depth >= heading && !anchorLinks?.exclude.includes(heading)
 </script>
diff --git a/dist/runtime/components/Prose/ProseH5.vue b/dist/runtime/components/Prose/ProseH5.vue
index 032f3e7073e5dfe45923b75cea1d69989265fe3f..45ca28e3ee884c81428e845ed31b5b01b60c19be 100644
--- a/dist/runtime/components/Prose/ProseH5.vue
+++ b/dist/runtime/components/Prose/ProseH5.vue
@@ -12,5 +12,6 @@ import { useRuntimeConfig } from '#imports'
 defineProps<{ id?: string }>()
 const heading = 5
 const { anchorLinks } = useRuntimeConfig().public.content
+// @ts-expect-error: https://github.com/nuxt/content/issues/1856
 const generate = anchorLinks?.depth >= heading && !anchorLinks?.exclude.includes(heading)
 </script>
diff --git a/dist/runtime/components/Prose/ProseH6.vue b/dist/runtime/components/Prose/ProseH6.vue
index 0abccfaae97bb46efd618f6386010a01ab557682..a9ec6d8d34dabfb53b335fd4984ce35a3ecbb4c0 100644
--- a/dist/runtime/components/Prose/ProseH6.vue
+++ b/dist/runtime/components/Prose/ProseH6.vue
@@ -12,5 +12,6 @@ import { useRuntimeConfig } from '#imports'
 defineProps<{ id?: string }>()
 const heading = 6
 const { anchorLinks } = useRuntimeConfig().public.content
+// @ts-expect-error: https://github.com/nuxt/content/issues/1856
 const generate = anchorLinks?.depth >= heading && !anchorLinks?.exclude.includes(heading)
 </script>
