diff --git a/dist/module.mjs b/dist/module.mjs
index c0aa42d5d0a2c231decd1bb8db68377fb6411596..7c159f69ae25806a879fe89777a7d5cd83465203 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -146,6 +146,12 @@ const module = defineNuxtModule({
         filename: "tailwind.config.mjs",
         getContents: () => `export default ${JSON.stringify(resolvedConfig, null, 2)}`
       });
+      // CHANGED: Implement https://github.com/nuxt-modules/tailwindcss/issues/533
+      addTemplate({
+        filename: "tailwind.config.cjs",
+        getContents: () => `module.export = ${JSON.stringify(resolvedConfig, null, 2)}`,
+        write: true
+      });
       addTemplate({
         filename: "tailwind.config.d.ts",
         getContents: () => 'declare const config: import("tailwindcss").Config\nexport { config as default }',
