diff --git a/dist/shared/nitro.7a525996.mjs b/dist/shared/nitro.7a525996.mjs
index 90c7e4a5618763b6e882de6417531c76089c6868..d2b31e6cc7a66e956fafefb6fcb5b56f194f4905 100644
--- a/dist/shared/nitro.7a525996.mjs
+++ b/dist/shared/nitro.7a525996.mjs
@@ -2546,6 +2546,11 @@ async function writeRoutes$1(nitro) {
     routes: [],
     navigationFallback: {
       rewrite: "/api/server"
+    },
+    // CHANGED: there is currently no way to extend the config (see https://github.com/unjs/nitro/discussions/420)
+    // but we need to provide global robots header (no idea why, otherwise its set to none)
+    "globalHeaders": {
+      "X-Robots-Tag": "index, follow, max-image-preview:large, max-snippet:-1, max-video-preview:-1"
     }
   };
   const routeFiles = nitro._prerenderedRoutes || [];
