diff --git a/dist/shared/nitro.96034a6e.mjs b/dist/shared/nitro.96034a6e.mjs
index 318eb6aaac9787d1d0301508b1bc284154f4d3ab..a353af04a5ad19d99fc3a3c3332e5ed6a5ccc13b 100644
--- a/dist/shared/nitro.96034a6e.mjs
+++ b/dist/shared/nitro.96034a6e.mjs
@@ -2464,6 +2464,11 @@ async function writeRoutes$1(nitro) {
     routes: [],
     navigationFallback: {
       rewrite: "/api/server"
+    },
+    // CHANGED: there is currently no way to extend the config (see https://github.com/unjs/nitro/discussions/420)
+    // but we need to provide global robots header (no idea why, otherwise its set to none)
+    "globalHeaders": {
+      "X-Robots-Tag": "index, follow, max-image-preview:large, max-snippet:-1, max-video-preview:-1"
     }
   };
   const routeFiles = nitro._prerenderedRoutes || [];
