diff --git a/dist/runtime/entries/azure-functions.mjs b/dist/runtime/entries/azure-functions.mjs
index 7437e5b196f983f4706d56c5820e664b4f1d4098..30251a23976e8a261778404b1e13561d3fa5c2e0 100644
--- a/dist/runtime/entries/azure-functions.mjs
+++ b/dist/runtime/entries/azure-functions.mjs
@@ -18,4 +18,5 @@ export async function handle(context, req) {
     headers: normalizeLambdaOutgoingHeaders(headers, true),
     body: body ? body.toString() : statusText
   };
+  context.error("Nitro response", context.res.cookies, context.res.headers, headers)
 }
diff --git a/dist/runtime/entries/azure.mjs b/dist/runtime/entries/azure.mjs
index 2ed72eb8e5f5f6b740d909bb12f62cf761599569..db36cee92c7638fc99e02f4ac9ff1f7b86eb2d99 100644
--- a/dist/runtime/entries/azure.mjs
+++ b/dist/runtime/entries/azure.mjs
@@ -25,4 +25,5 @@ export async function handle(context, req) {
     headers: normalizeLambdaOutgoingHeaders(headers, true),
     body: body ? body.toString() : statusText
   };
+  context.log.error("Nitro response", context.res.cookies, context.res.headers, headers)
 }
diff --git a/dist/runtime/utils.azure.mjs b/dist/runtime/utils.azure.mjs
index d0e27bd1b36683f7834ed097c07702429da3c152..b078579627e739876113b19ab5fa294c0413e038 100644
--- a/dist/runtime/utils.azure.mjs
+++ b/dist/runtime/utils.azure.mjs
@@ -7,7 +7,17 @@ export function getAzureParsedCookiesFromHeaders(headers) {
     return [];
   }
   const cookies = splitCookiesString(joinHeaders(c)).map(
-    (cookie) => parse(cookie)
+    (cookie) => {
+        const entries = Object.entries(parse(cookie))
+        if (entries.length >= 1) {
+          const [entry, ...rest] = Object.entries(cookie)
+          return {
+            name: entry[0],
+            value: entry[1],
+            ...Object.fromEntries(rest)
+          }
+        }
+      }
   );
   return cookies;
 }
