diff --git a/dist/runtime/entries/azure.mjs b/dist/runtime/entries/azure.mjs
index fe72a0be7cd3984a638234280b409a7ca803b5d0..b11f6ebbfdd8520fbefdf3cbe6bd391a07893915 100644
--- a/dist/runtime/entries/azure.mjs
+++ b/dist/runtime/entries/azure.mjs
@@ -3,6 +3,8 @@ import { parseURL } from "ufo";
 import { nitroApp } from "../app.mjs";
 import { getAzureParsedCookiesFromHeaders } from "../utils.azure.mjs";
 import { normalizeLambdaOutgoingHeaders } from "../utils.lambda.mjs";
+import { createConsola } from "consola";
+
 export async function handle(context, req) {
   let url;
   if (req.headers["x-ms-original-url"]) {
@@ -11,6 +13,25 @@ export async function handle(context, req) {
   } else {
     url = "/api/" + (req.params.url || "");
   }
+  const _getLogFn = (level) => {
+    if (level < 1) {
+      return context.log.error;
+    }
+    if (level === 1) {
+      return context.log.warn;
+    }
+    return context.log;
+  }
+  const consola = createConsola({
+    reporters: [
+      {
+        log: (logObj) => {
+          _getLogFn(logObj.level)(logObj.args);
+        },
+      },
+    ],
+  });
+  consola.wrapConsole();
   const { body, status, headers } = await nitroApp.localCall({
     url,
     headers: req.headers,
@@ -18,6 +39,7 @@ export async function handle(context, req) {
     // https://github.com/Azure/azure-functions-host/issues/293
     body: req.rawBody
   });
+  consola.restoreConsole();
   context.res = {
     status,
     cookies: getAzureParsedCookiesFromHeaders(headers),
