diff --git a/dist/shared/nitro.00419e14.mjs b/dist/shared/nitro.00419e14.mjs
index 61590df3b47b50bd4b78113c23741daddaef0e8b..87c6836e9cc6ccdef0756b4e63df404f3ad35aa2 100644
--- a/dist/shared/nitro.00419e14.mjs
+++ b/dist/shared/nitro.00419e14.mjs
@@ -2197,6 +2197,10 @@ const NitroDefaults = {
 };
 async function loadOptions(userConfig = {}) {
   let preset = process.env.NITRO_PRESET || userConfig.preset || detectTarget() || "node-server";
+  // CHANGED: Workaround for prerendering in azure preset
+  if (userConfig.preset === "nitro-prerender") {
+    preset = "nitro-prerender"
+  }
   if (userConfig.dev) {
     preset = "nitro-dev";
   }
