diff --git a/dist/runtime/server/robots-middleware.mjs b/dist/runtime/server/robots-middleware.mjs
index a95ac80f5664251144603d23c85fd2985d340a17..1d41483adaa5f0cfcd3bba1738adfbff920c697f 100644
--- a/dist/runtime/server/robots-middleware.mjs
+++ b/dist/runtime/server/robots-middleware.mjs
@@ -1,12 +1,9 @@
 import { defineEventHandler, setHeader } from "h3";
-import { indexable, robotsDisabledValue } from "#nuxt-simple-robots/config";
+import { indexable, robotsDisabledValue, robotsEnabledValue } from "#nuxt-simple-robots/config";
 import { getRouteRules } from "#internal/nitro";
 export default defineEventHandler((event) => {
   if (event.path === "/robots.txt")
     return;
-  const routeRules = getRouteRules(event);
-  if (typeof routeRules.robots === "string")
-    setHeader(event, "X-Robots-Tag", routeRules.robots);
-  else if (routeRules.index === false || indexable === false)
-    setHeader(event, "X-Robots-Tag", robotsDisabledValue);
+  // CHANGED: Otherwise x-robots-tag is set to none
+  setHeader(event, "X-Robots-Tag", robotsEnabledValue);
 });
