diff --git a/lib/server/redis-db.js b/lib/server/redis-db.js
index 35583cc92b5d0b26ac9e8373a2808b133c188552..2eea5dfa3b11ca76891dc11cd8e936533fa1d28b 100644
--- a/lib/server/redis-db.js
+++ b/lib/server/redis-db.js
@@ -28,7 +28,8 @@ class RedisDb {
  *
  * The server contains a log of logic. It only feels natural to split it into multiple files
  */
-['./strings', './keys', './hash', './set', './list.js', './sortedset']
-  .forEach((lib) => Object.assign(RedisDb.prototype, require(lib)));
+// CHANGED: Don't need those, and they throw module not found errors
+//['./strings', './keys', './hash', './set', './list.js', './sortedset']
+//  .forEach((lib) => Object.assign(RedisDb.prototype, require(lib)));
 
 module.exports = RedisDb;
