diff --git a/node_modules/@apollo/client/utilities/globals/globals.cjs b/node_modules/@apollo/client/utilities/globals/globals.cjs
index 93da0b5..086d935 100644
--- a/node_modules/@apollo/client/utilities/globals/globals.cjs
+++ b/node_modules/@apollo/client/utilities/globals/globals.cjs
@@ -3,7 +3,7 @@
 Object.defineProperty(exports, '__esModule', { value: true });
 
 var tsInvariant = require('ts-invariant');
-var process$1 = require('ts-invariant/process');
+var process$1 = require('ts-invariant/process/main.js');
 var graphql = require('graphql');
 
 function maybe(thunk) {
