diff --git a/node_modules/@nuxt/bridge/dist/chunks/module.mjs b/node_modules/@nuxt/bridge/dist/chunks/module.mjs
index 7802bf8..137a469 100644
--- a/node_modules/@nuxt/bridge/dist/chunks/module.mjs
+++ b/node_modules/@nuxt/bridge/dist/chunks/module.mjs
@@ -127,8 +127,9 @@ const distDir$1 = join(pkgDir, "dist");
 
 function setupNitroBridge() {
   const nuxt = useNuxt();
-  if (!nuxt.options.dev && nuxt.options.target === "static" && !nuxt.options._export && !nuxt.options._legacyGenerate) {
-    throw new Error("[nitro] Please use `nuxt generate` for static target");
+  console.log(nuxt.options)
+  if (!nuxt.options.dev && !nuxt.options._build && nuxt.options.target === "static" && !nuxt.options._export && !nuxt.options._legacyGenerate) {
+    //throw new Error("[nitro] Please use `nuxt generate` for static target");
   }
   nuxt.options.build.loadingScreen = false;
   nuxt.options.build.indicator = false;
