diff --git a/node_modules/@nuxt/bridge/dist/chunks/module.mjs b/node_modules/@nuxt/bridge/dist/chunks/module.mjs
index 7e5831c..a64002a 100644
--- a/node_modules/@nuxt/bridge/dist/chunks/module.mjs
+++ b/node_modules/@nuxt/bridge/dist/chunks/module.mjs
@@ -128,7 +128,7 @@ const distDir$1 = join(pkgDir, "dist");
 function setupNitroBridge() {
   const nuxt = useNuxt();
   if (!nuxt.options.dev && nuxt.options.target === "static" && !nuxt.options._prepare && !nuxt.options._export && !nuxt.options._legacyGenerate) {
-    throw new Error("[nitro] Please use `nuxt generate` for static target");
+    // throw new Error("[nitro] Please use `nuxt generate` for static target");
   }
   nuxt.options.build.loadingScreen = false;
   nuxt.options.build.indicator = false;
@@ -245,7 +245,7 @@ function setupNitroBridge() {
   });
   nuxt.hook("generate:extendRoutes", async () => {
     await build(nitroDevContext);
-    await nuxt.server.reload();
+    //await nuxt.server.reload();
   });
   nuxt.hook("generate:done", async () => {
     await nuxt.server.close();
diff --git a/node_modules/@nuxt/bridge/dist/chunks/module2.mjs b/node_modules/@nuxt/bridge/dist/chunks/module2.mjs
index 41e88bd..3581d70 100644
--- a/node_modules/@nuxt/bridge/dist/chunks/module2.mjs
+++ b/node_modules/@nuxt/bridge/dist/chunks/module2.mjs
@@ -2,6 +2,7 @@ import consola from 'consola';
 import { defineNuxtModule, addPluginTemplate } from '@nuxt/kit';
 import hash from 'hash-sum';
 import { resolve } from 'pathe';
+import { build } from 'vite';
 
 const version = "3.0.0-27284664.232fdd8";
 
@@ -151,8 +152,10 @@ const module = defineNuxtModule({
     }
     addPluginTemplate(middlewareTemplate);
     nuxt.hook("builder:prepared", async (builder) => {
-      builder.bundleBuilder.close();
-      delete builder.bundleBuilder;
+      if (builder.bundleBuilder && builder.bundleBuilder.close) {
+        builder.bundleBuilder.close();
+        delete builder.bundleBuilder;
+      }
       const { ViteBuilder } = await import('./vite.mjs');
       builder.bundleBuilder = new ViteBuilder(builder);
     });
diff --git a/node_modules/@nuxt/bridge/dist/runtime/app.d.ts b/node_modules/@nuxt/bridge/dist/runtime/app.d.ts
index c7dff20..7644e20 100644
--- a/node_modules/@nuxt/bridge/dist/runtime/app.d.ts
+++ b/node_modules/@nuxt/bridge/dist/runtime/app.d.ts
@@ -1,7 +1,8 @@
 import type { Hookable } from 'hookable';
-import type { Vue } from 'vue/types/vue';
+import type { VueConstructor } from 'vue/types/vue';
 import type { ComponentOptions } from 'vue';
 import { defineComponent } from './composables';
+import { Plugin } from '@nuxt/types';
 export declare const isVue2 = true;
 export declare const isVue3 = false;
 export declare const defineNuxtComponent: typeof defineComponent;
@@ -24,7 +25,7 @@ export interface VueAppCompat {
 export interface NuxtAppCompat {
     nuxt2Context: Vue;
     vue2App: ComponentOptions<Vue>;
-    vueApp: VueAppCompat;
+    vueApp: VueConstructor;
     globalName: string;
     hooks: Hookable<RuntimeNuxtHooks>;
     hook: NuxtAppCompat['hooks']['hook'];
@@ -40,5 +41,5 @@ export interface Context {
     $_nuxtApp: NuxtAppCompat;
 }
 export declare const setNuxtAppInstance: (nuxt: NuxtAppCompat | null) => void;
-export declare const defineNuxtPlugin: (plugin: any) => (ctx: Context) => void;
+export declare function defineNuxtPlugin (plugin: (nuxtApp: NuxtAppCompat) => void): (ctx: Context) => void;
 export declare const useNuxtApp: () => any;
diff --git a/node_modules/@nuxt/bridge/dist/runtime/app.plugin.mjs b/node_modules/@nuxt/bridge/dist/runtime/app.plugin.mjs
index 522b5e0..6515e77 100644
--- a/node_modules/@nuxt/bridge/dist/runtime/app.plugin.mjs
+++ b/node_modules/@nuxt/bridge/dist/runtime/app.plugin.mjs
@@ -23,7 +23,8 @@ export default (ctx, inject) => {
     vueApp: {
       component: Vue.component.bind(Vue),
       config: {
-        globalProperties: {}
+        globalProperties: {},
+        optionMergeStrategies: Vue.config.optionMergeStrategies
       },
       directive: Vue.directive.bind(Vue),
       mixin: Vue.mixin.bind(Vue),
@@ -33,7 +34,8 @@ export default (ctx, inject) => {
       use (vuePlugin) {
         vuePlugin.install(this)
       },
-      version: Vue.version
+      version: Vue.version,
+      prototype: {}
     },
     provide: inject,
     globalName: 'nuxt',
