diff --git a/node_modules/@nuxt/nitro/dist/index.mjs b/node_modules/@nuxt/nitro/dist/index.mjs
index 9fed817..ff22217 100644
--- a/node_modules/@nuxt/nitro/dist/index.mjs
+++ b/node_modules/@nuxt/nitro/dist/index.mjs
@@ -1708,6 +1708,7 @@ function initWorker(filename) {
     });
     worker.on("error", (err) => {
       err.message = "[worker] " + err.message;
+      console.error(err);
       reject(err);
     });
     worker.on("message", (event) => {
