diff --git a/node_modules/@vue/apollo-composable/dist/index.js b/node_modules/@vue/apollo-composable/dist/index.js
index aeedcb4..294b954 100644
--- a/node_modules/@vue/apollo-composable/dist/index.js
+++ b/node_modules/@vue/apollo-composable/dist/index.js
@@ -234,7 +234,7 @@ function trackSubscription(loading) {
 }
 
 // src/util/toApolloError.ts
-var import_core = __toModule(require("@apollo/client/core"));
+var import_core = __toModule(require("@apollo/client/core/core.cjs"));
 function toApolloError(error) {
   if (!(error instanceof Error)) {
     return new import_core.ApolloError({
