diff --git a/node_modules/@vue/apollo-util/dist/errorLog.js b/node_modules/@vue/apollo-util/dist/errorLog.js
index 0ba1121..78c88e4 100644
--- a/node_modules/@vue/apollo-util/dist/errorLog.js
+++ b/node_modules/@vue/apollo-util/dist/errorLog.js
@@ -1,7 +1,7 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.logErrorMessages = exports.getErrorMessages = void 0;
-var printer_1 = require("graphql/language/printer");
+var printer_1 = require("graphql/language/printer.js");
 function getErrorMessages(error) {
     var messages = [];
     var graphQLErrors = error.graphQLErrors, networkError = error.networkError;
