diff --git a/node_modules/h3/dist/index.cjs b/node_modules/h3/dist/index.cjs
index e72fc41..e8dec25 100644
--- a/node_modules/h3/dist/index.cjs
+++ b/node_modules/h3/dist/index.cjs
@@ -243,7 +243,8 @@ function createError(input) {
   if (isError(input)) {
     return input;
   }
-  const err = new H3Error(input.message ?? input.statusMessage, input.cause ? { cause: input.cause } : void 0);
+  // CHANGED: Workaround for https://github.com/nuxt-community/storybook/issues/377
+  const err = new H3Error(input.message !== null ? input.message : input.statusMessage, input.cause ? { cause: input.cause } : void 0);
   if ("stack" in input) {
     try {
       Object.defineProperty(err, "stack", { get() {
