diff --git a/node_modules/h3/dist/index.cjs b/node_modules/h3/dist/index.cjs
index 9ed930c..247f529 100644
--- a/node_modules/h3/dist/index.cjs
+++ b/node_modules/h3/dist/index.cjs
@@ -198,7 +198,8 @@ function createError(input) {
   if (input instanceof H3Error) {
     return input;
   }
-  const err = new H3Error(input.message ?? input.statusMessage);
+  // CHANGED: Workaround for https://github.com/nuxt-community/storybook/issues/377
+  const err = new H3Error(input.message !== null ? input.message : input.statusMessage);
   if (input.statusCode) {
     err.statusCode = input.statusCode;
   }
