diff --git a/build/index.js b/build/index.js
index 1fe094a3185252ef3b4d2201055a95f982833f72..60e9fc888f34a8c1027decc4856c047414fd6f9c 100644
--- a/build/index.js
+++ b/build/index.js
@@ -2489,7 +2489,7 @@ ${u.length>0?`${u.join(`
 `;let n=`import * as process from 'node:process'
 import * as path from 'node:path'
 `;return e==="esm"&&(n+=`import { fileURLToPath } from 'node:url'
-const __dirname = path.dirname(fileURLToPath(import.meta.url))
+const __dirname = '/'
 `),n}var S0t=`/**
  * This file exports various common sort, input & filter types that are not directly linked to a particular model.
  *
