diff --git a/build/index.js b/build/index.js
index 53b71b5f8bfad956bd73dee28747811d0eb191fa..63334216050baf185032ab3fbac611aa7f6c6766 100644
--- a/build/index.js
+++ b/build/index.js
@@ -2480,7 +2480,7 @@ ${u.length>0?`${u.join(`
 `;let n=`import * as process from 'node:process'
 import * as path from 'node:path'
 `;return e==="esm"&&(n+=`import { fileURLToPath } from 'node:url'
-globalThis['__dirname'] = path.dirname(fileURLToPath(import.meta.url))
+globalThis['__dirname'] = "DOESNOTEXIST"
 `),n}var S0t=`/*
  * This file exports various common sort, input & filter types that are not directly linked to a particular model.
  *
