diff --git a/build/index.js b/build/index.js
index 38b86db5f81e33f59a58d7118fe26dd853043be4..386e6cc11860c79657bb3fe4c683bee5d625711c 100755
--- a/build/index.js
+++ b/build/index.js
@@ -3179,7 +3179,7 @@ ${Z$(e).join(`
 `;let n=`import * as process from 'node:process'
 import * as path from 'node:path'
 `;return r==="esm"&&(n+=`import { fileURLToPath } from 'node:url'
-globalThis['__dirname'] = path.dirname(fileURLToPath(import.meta.url))
+globalThis['__dirname'] = "DOESNOTEXIST"
 `),n}var FLt=`/*
  * This file exports various common sort, input & filter types that are not directly linked to a particular model.
  *
