diff --git a/node_modules/redis-mock/lib/server/redis-db.js b/node_modules/redis-mock/lib/server/redis-db.js
index 35583cc..2eea5df 100644
--- a/node_modules/redis-mock/lib/server/redis-db.js
+++ b/node_modules/redis-mock/lib/server/redis-db.js
@@ -28,7 +28,8 @@ class RedisDb {
  *
  * The server contains a log of logic. It only feels natural to split it into multiple files
  */
-['./strings', './keys', './hash', './set', './list.js', './sortedset']
-  .forEach((lib) => Object.assign(RedisDb.prototype, require(lib)));
+// CHANGED: Don't need those, and they throw module not found errors
+//['./strings', './keys', './hash', './set', './list.js', './sortedset']
+//  .forEach((lib) => Object.assign(RedisDb.prototype, require(lib)));
 
 module.exports = RedisDb;
