diff --git a/node_modules/vue-virtual-scroll-list/dist/index.js b/node_modules/vue-virtual-scroll-list/dist/index.js
index e5856c1..41d04b5 100644
--- a/node_modules/vue-virtual-scroll-list/dist/index.js
+++ b/node_modules/vue-virtual-scroll-list/dist/index.js
@@ -4,13 +4,9 @@
  * https://github.com/tangbc/vue-virtual-scroll-list#readme
  */
 
-(function (global, factory) {
-  typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory(require('vue')) :
-  typeof define === 'function' && define.amd ? define(['vue'], factory) :
-  (global = global || self, global.VirtualList = factory(global.Vue));
-}(this, (function (Vue) { 'use strict';
+  // CHANGE: Workaround for https://github.com/tangbc/vue-virtual-scroll-list/issues/367 and https://github.com/nuxt/framework/issues/2440
+  import Vue from "vue";
 
-  Vue = Vue && Object.prototype.hasOwnProperty.call(Vue, 'default') ? Vue['default'] : Vue;
 
   function _classCallCheck(instance, Constructor) {
     if (!(instance instanceof Constructor)) {
@@ -1054,6 +1050,5 @@
     }
   });
 
-  return VirtualList;
+  export default VirtualList;
 
-})));
