diff --git a/node_modules/vue-tailwind/dist/components.js b/node_modules/vue-tailwind/dist/components.js
index 74aa8f9..5f942ba 100644
--- a/node_modules/vue-tailwind/dist/components.js
+++ b/node_modules/vue-tailwind/dist/components.js
@@ -1,21 +1,17 @@
-(function (global, factory) {
-    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('vue'), require('lodash.get'), require('lodash.isequal'), require('lodash.map'), require('lodash.clonedeep'), require('lodash.intersection'), require('body-scroll-lock'), require('lodash.range'), require('lodash.kebabcase'), require('lodash.pick'), require('lodash.merge')) :
-    typeof define === 'function' && define.amd ? define(['exports', 'vue', 'lodash.get', 'lodash.isequal', 'lodash.map', 'lodash.clonedeep', 'lodash.intersection', 'body-scroll-lock', 'lodash.range', 'lodash.kebabcase', 'lodash.pick', 'lodash.merge'], factory) :
-    (global = typeof globalThis !== 'undefined' ? globalThis : global || self, factory(global.VueTailwind = {}, global.Vue, global.get, global.isEqual, global.map, global.cloneDeep, global.intersection, global.bodyScrollLock, global.range, global.kebabCase, global.pick, global.merge));
-}(this, (function (exports, Vue, get, isEqual, map, cloneDeep, intersection, bodyScrollLock, range, kebabCase, pick, merge) { 'use strict';
+    // CHANGE: Workaround for https://github.com/nuxt/framework/issues/2440 and https://github.com/alfonsobries/vue-tailwind/issues/236
 
-    function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }
-
-    var Vue__default = /*#__PURE__*/_interopDefaultLegacy(Vue);
-    var get__default = /*#__PURE__*/_interopDefaultLegacy(get);
-    var isEqual__default = /*#__PURE__*/_interopDefaultLegacy(isEqual);
-    var map__default = /*#__PURE__*/_interopDefaultLegacy(map);
-    var cloneDeep__default = /*#__PURE__*/_interopDefaultLegacy(cloneDeep);
-    var intersection__default = /*#__PURE__*/_interopDefaultLegacy(intersection);
-    var range__default = /*#__PURE__*/_interopDefaultLegacy(range);
-    var kebabCase__default = /*#__PURE__*/_interopDefaultLegacy(kebabCase);
-    var pick__default = /*#__PURE__*/_interopDefaultLegacy(pick);
-    var merge__default = /*#__PURE__*/_interopDefaultLegacy(merge);
+    import Vue from 'vue';
+    var Vue__default = { default: Vue};
+    import * as get__default from 'lodash.get';
+    import * as isEqual__default from 'lodash.isequal';
+    import * as map__default from 'lodash.map';
+    import * as cloneDeep__default from 'lodash.clonedeep';
+    import * as intersection__default from 'lodash.intersection';
+    import * as range__default from 'lodash.range';
+    import * as kebabCase__default from 'lodash.kebabcase';
+    import * as pick__default from 'lodash.pick';
+    import * as merge__default from 'lodash.merge';
+    import * as bodyScrollLock from 'body-scroll-lock';
 
     const mergeClasses = (classesA, classesB) => {
         let a = classesA;
@@ -10120,28 +10116,25 @@
         },
     });
 
-    exports.TAlert = TAlert;
-    exports.TButton = TButton;
-    exports.TCard = TCard;
-    exports.TCheckbox = TCheckbox;
-    exports.TCheckboxGroup = TCheckboxGroup;
-    exports.TDatepicker = TDatepicker;
-    exports.TDialog = TDialog;
-    exports.TDropdown = TDropdown;
-    exports.TInput = TInput;
-    exports.TInputGroup = TInputGroup;
-    exports.TModal = TModal;
-    exports.TPagination = TPagination;
-    exports.TRadio = TRadio;
-    exports.TRadioGroup = TRadioGroup;
-    exports.TRichSelect = TRichSelect;
-    exports.TSelect = TSelect;
-    exports.TTable = TTable;
-    exports.TTag = TTag;
-    exports.TTextarea = TTextarea;
-    exports.TToggle = TToggle;
-
-    Object.defineProperty(exports, '__esModule', { value: true });
+    export {TAlert};
+    export {TButton};
+    export {TCard};
+    export {TCheckbox};
+    export {TCheckboxGroup};
+    export {TDatepicker};
+    export {TDialog};
+    export {TDropdown};
+    export {TInput};
+    export {TInputGroup};
+    export {TModal};
+    export {TPagination};
+    export {TRadio};
+    export {TRadioGroup};
+    export {TRichSelect};
+    export {TSelect};
+    export {TTable};
+    export {TTag};
+    export {TTextarea};
+    export {TToggle};
 
-})));
 //# sourceMappingURL=components.js.map
diff --git a/node_modules/vue-tailwind/dist/vue-tailwind.js b/node_modules/vue-tailwind/dist/vue-tailwind.js
index 0911f3b..e1167d2 100644
--- a/node_modules/vue-tailwind/dist/vue-tailwind.js
+++ b/node_modules/vue-tailwind/dist/vue-tailwind.js
@@ -1,14 +1,9 @@
-(function (global, factory) {
-    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('vue'), require('lodash.get'), require('body-scroll-lock'), require('lodash.map')) :
-    typeof define === 'function' && define.amd ? define(['exports', 'vue', 'lodash.get', 'body-scroll-lock', 'lodash.map'], factory) :
-    (global = typeof globalThis !== 'undefined' ? globalThis : global || self, factory(global.VueTailwind = {}, global.Vue, global.get, global.bodyScrollLock, global.map));
-}(this, (function (exports, Vue, get, bodyScrollLock, map) { 'use strict';
-
-    function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }
-
-    var Vue__default = /*#__PURE__*/_interopDefaultLegacy(Vue);
-    var get__default = /*#__PURE__*/_interopDefaultLegacy(get);
-    var map__default = /*#__PURE__*/_interopDefaultLegacy(map);
+    // CHANGE: Workaround for https://github.com/nuxt/framework/issues/2440 and https://github.com/alfonsobries/vue-tailwind/issues/236
+    import Vue from 'vue';
+    var Vue__default = { default: Vue};
+    import * as get__default from 'lodash.get';
+    import * as map__default from 'lodash.map';
+    import * as bodyScrollLock from 'body-scroll-lock';
 
     /* eslint-disable no-shadow */
     var HideReason;
@@ -1986,10 +1981,7 @@
         install,
     };
 
-    exports.BaseComponent = Component;
-    exports.default = plugin;
-
-    Object.defineProperty(exports, '__esModule', { value: true });
+    //    exports.BaseComponent = Component;
+    export default plugin;
 
-})));
 //# sourceMappingURL=vue-tailwind.js.map
